pkgs/applications/audio/audacity/default.nix

```diff
@@ -166,16 +169,31 @@ stdenv.mkDerivation rec {
   ];
 
   cmakeFlags = [
-    "-Daudacity_use_ffmpeg=linked"
+    "-DAUDACITY_REV_LONG=nixpkgs"
+    "-DAUDACITY_REV_TIME=nixpkgs"
     "-DDISABLE_DYNAMIC_LOADING_FFMPEG=ON"
+    "-Daudacity_conan_enabled=Off"
+    "-Daudacity_use_ffmpeg=loaded"
   ];
 
   doCheck = false; # Test fails
 
+  # Replace audacity's wrapper, to:
+  # - put it in the right place, it shouldn't be in "$out/audacity"
+  # - Add the ffmpeg dynamic dependency
+  postInstall = lib.optionalString stdenv.isLinux ''
+    rm "$out/audacity"
```

Comment:
Is there a setting in here that'd move this whole $out/audacity dir?

https://github.com/audacity/audacity/blob/master/CMakeLists.txt#L370-L378

Also is it safe to just delete it?

Reply:
@Infinisil cc

Reply:
There's no setting to not generate that. The script comes from [here](https://github.com/audacity/audacity/blob/master/linux/audacity.sh), it's pretty simple, we just create our own version with the ffmpeg dependency added.

Reply:
I see.

I guess we could patch this out here https://github.com/audacity/audacity/blob/dc2bdfe1a5695c161644001b06c565dd465d6ce6/src/CMakeLists.txt#L1511-L1513
but rm looks fine

Maybe `Replace audacity's wrapper, to:` -> `Replace audacity's existing wrapper, to: